client.lua

```diff
@@ -56,6 +56,14 @@ local function DrawText3D(coords, text)
     ClearDrawOrigin()
 end
 
+lib.callback.register('qbx-storerobbery:client:GetCurrentTime', function()
```

Comment:
nit; GetCurrentTime -> getCurrentTime

Reply:
The body of this callback function doesn't match the behavior. It should be called getAlertChance.

Reply:
Why does this callback function even need to exist on the client? Can't the server just do this?

Reply:
You are right, it should be getAlertChance. So for event names we also apply camelCase?

It's a callback on clientside because GetClockHours() only works on client side: https://docs.fivem.net/natives/?_0x25223CA6B4D20B7F and I am looking at getting the actual in-game time to make sure if it's night to provide a lower chance of alert.

Reply:
If time is synced between players there should be an export to get the time. Have you checked weathersync?

Reply:
Nothing there:

-- EXPORTS
exports('nextWeatherStage', nextWeatherStage)
exports('setWeather', setWeather)
exports('setTime', setTime)
exports('setBlackout', setBlackout)
exports('setTimeFreeze', setTimeFreeze)
exports('setDynamicWeather', setDynamicWeather)
exports('getBlackoutState', function() return blackout end)
exports('getTimeFreezeState', function() return freezeTime end)
exports('getWeatherState', function() return CurrentWeather end)
exports('getDynamicWeather', function() return Config.DynamicWeather end)

Reply:
What about introducing one then?